homeassistant/helpers/device_registry.py

```diff
@@ -41,10 +41,6 @@
 CONNECTION_UPNP = "upnp"
 CONNECTION_ZIGBEE = "zigbee"
 
-DISABLED_CONFIG_ENTRY = "config_entry"
-DISABLED_INTEGRATION = "integration"
-DISABLED_USER = "user"
```

Comment:
This is breaking for custom integrations.

Reply:
See #60421